frontend/webadmin/modules/frontend/src/main/resources/org/ovirt/engine/ui/frontend/AppErrors_de_DE.properties

```diff
@@ -652,7 +649,6 @@ ACTION_TYPE_FAILED_UNSUPPORTED_OVF=Kann ${type} nicht ${action}. Kein OVF f\u00F
 ACTION_TYPE_FAILED_UPDATE_OF_FAILOVER_PROFILE_IS_NOT_SUPPORTED=Kann nicht ${action} ${type}. Profil wird als Failover f\u00FCr ${ENTITIES_USING_FAILOVER} ausgew\u00E4hlt.
 ACTION_TYPE_FAILED_USERNAME_MUST_BE_SPECIFIED=Kann ${type} nicht ${action}. Es wurde kein Benutzername angegeben.
 ACTION_TYPE_FAILED_USER_PROFILE_MULTIPLE_SSH_KEYS=Kann nicht ${action} ${type}. Mehrere \u00F6ffentliche SSH-Schl\u00FCssel werden nicht unterst\u00FCtzt.
-ACTION_TYPE_FAILED_USER_PROFILE_PROPERTY_ALREADY_EXISTS=Kann nicht ${action} ${type}. Benutzerprofil-Eigenschaft namens ${propName} ist bereits vorhanden. Eigenschaftstyp ist ${propType} und ID\=${propId}.
```

Comment:
This was invalidated only because in English text a space was removed at the end of the message.
See https://github.com/oVirt/ovirt-engine/commit/5cfed6548922db9fb7f23c170ef5bf25074754f4

Reply:
Fixed in Zanata